input/pagecontent/sexgender.md

```diff
@@ -0,0 +1,336 @@
+          "valueCodeableConcept" : {
+            "coding" : [
+              {
+                "system" : "http://loinc.org",
+                "code" : "LA29519-8",
+                "display" : "she/her/her/hers/herself"
+              }
+            ]
+          }
+        },
+        {
+          "url" : "period",
+          "valuePeriod" : {
+            "start" : "2001-05-06"
+          }
+        }
+      ]
+    },
+  ...
+}  
+~~~
+
+Example: Patient resource with prefer not to answer for pronouns
+~~~
+{
+  "resourceType" : "Patient",
+    ...
+    {
+      "url" : "http://hl7.org/fhir/StructureDefinition/individual-pronouns",
+      "extension" : [
+        {
+          "url" : "value",
+          "valueCodeableConcept" : {
+            "coding" : [
+              {
+                "system" : "http://terminology.hl7.org/CodeSystem/data-absent-reason",
+                "code" : "asked-declined",
+              }
+            ]
+          }
+        }
+      ]
+    },
+  ...
+}  
+~~~
+
+
+### Gender Identity (GI)
+AU Base supports representation and exchange of the Gender Identity data element (as defined in the [HL7 Cross Paradigm Implementation Guide: Gender Harmony - Sex and Gender Representation, Edition 1](https://hl7.org/xprod/ig/uv/gender-harmony/informative1/)) and aligned to [Australian Bureau of Statics Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) in:
+- [AU Base Patient](StructureDefinition-au-patient.html)
+- [AU Base Practitioner](StructureDefinition-au-practitioner.html)
+- [AU Base Related Person](StructureDefinition-au-relatedperson.html)
+
+Gender Identity is represented with the [Individual Gender Identity](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) extension. In AU Base the value element of the [Individual Gender Identity](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) extension is constrained to be [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) value set ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)).
+
+When populating the value element of the [Individual Gender Identity](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) extension:
+  - *Man or male* may be represented by sending the SNOMED CT code 446151000124109\|Identifies as male gender\|
+  - *Woman or female* may be represented by sending the SNOMED CT code 446141000124107\|Identifies as female gender\|
+  - *Non-binary* may be represented by sending the SNOMED CT code 33791000087105\|Identifies as nonbinary gender\|
+  - *[I/They] use a different term (please specify)*  may be represented by sending only `text` and no `coding`
+  - *Prefer not to answer* may be represented by sending the code "asked-declined"
```

Comment:
Suggest: ... sending the **Data Absent Reason** code “asked-declined”

Reply:
fixed

Reply:
Confirmed fix